docs/mp/guides/04_health.adoc

```diff
@@ -529,14 +528,14 @@ kubectl delete -f ./health.yaml
 ----
 
 === Summary
-This guide demonstrated how to use health-check in a Helidon MP application as follows:
+This guide demonstrated how to use health checks in a Helidon MP application as follows:
 
-* Access the default health-check
+* Access the default health check
 * Create and use custom readiness and liveness checks
-* Customize the health-check root path and port
-* Integrate Helidon health-check with Kubernetes
+* Customize the health check root path and port
+* Integrate Helidon health check with Kubernetes
```

Comment:
Maybe either plural "Helidon health checks" or "Helidon health check API" (taken from the earlier section text).